.github/workflows/master_deploy.yml

```diff
@@ -19,7 +19,8 @@ jobs:
               run: npm install
             - name: Deploy Application to GH Pages
               run: |
-                  git config user.email ${{ secrets.EMAIL }}
                   git config user.name ${{ secrets.USERNAME }}
```

Comment:
Why did you change this file? This shouldn't be changed, please revert your changes to this file.